### PR TITLE
This should clean up the JDatabaseTest file better.

### PR DESCRIPTION
**Pull request by Paladin**
Added different config file to JDatabaseMySQLTest to allow an independent decision
by the tester on whether these tests should be run. It now requires config_mysql.php
to contain the JMySQLTestConfig class to be present or the tests will be skipped.

### CODE REVIEW

File: tests/suite/joomla/database/database/JDatabaseMySQLTest.php [hunk 1]
```diff
@@ -59,7 +59,12 @@ class JDatabaseMySQLTest extends JoomlaDatabaseTestCase
 	 */
 	protected function setUp()
 	{
-		$config = new JTestConfig;
+		@include_once JPATH_TESTS . '/config_mysql.php';
+		if (class_exists('JMySQLTestConfig')) {
+			$config = new JTestConfig;
```
Comment (LouisLandry) [inline tests/suite/joomla/database/database/JDatabaseMySQLTest.php:64]: Shouldn't this be instantiating a new JMySQLTestConfig ?

Comment (Paladin) [inline tests/suite/joomla/database/database/JDatabaseMySQLTest.php:64]: Good catch. I should have caught this when I changed it to separate db testing configs. I'll close, fix and resubmit.

Comment (Paladin) [inline tests/suite/joomla/database/database/JDatabaseMySQLTest.php:64]: Let me rephrase that. Please close this pull request (can't seem to close it myself, oddly enough). I've resubmitted this as pull request #45.

**Comment by LouisLandry** [comment]
Cool deal.  Thx.
